src/libxmljs.cc

```diff
@@ -97,9 +98,53 @@ char* xmlMemoryStrdupWrap(const char* str)
     return res;
 }
 
-// callback function for `xmlDeregisterNodeDefault`
+void deregisterNsList(xmlNs* ns)
+{
+    while (ns != NULL) {
+        if (ns->_private != NULL) {
+            XmlNamespace* wrapper = static_cast<XmlNamespace*>(ns->_private);
+            wrapper->xml_obj = NULL;
+            ns->_private = NULL;
+        }
+        ns = ns->next;
+    }
+}
+
+void deregisterNodeNamespaces(xmlNode* xml_obj)
+{
+    xmlNs *ns = NULL;
```

Comment:
Inconsistent pointer placement. Other locations have the pointer next to the var name.

Maybe adding: http://clang.llvm.org/docs/ClangFormat.html to the project would be nice?